src/Symfony/Component/Process/PhpExecutableFinder.php

```diff
@@ -35,14 +35,17 @@ public function __construct()
      */
     public function find($includeArgs = true)
     {
+        $args = $this->findArguments();
+        $args = $includeArgs && $args ? ' '.implode(' ', $args) : '';
```

Comment:
I agree with @xabbuh. This change appears as ``Remove tmp addition of zend-stdlib`` in the history